packages/load-script/src/types/index.d.ts

```diff
@@ -1,4 +1,8 @@
-export function loadScript( url: string, callback: () => void ): undefined;
+export function loadScript(
+	url: string,
+	callback: () => void,
+	args?: Record< string, string >
+): undefined;
 export function loadScript( url: string ): Promise< void >;
 
 export function loadjQueryDependentScript( url: string, callback: () => void ): undefined;
```

Comment:
Did we mean to update this type as well, since we added `args` to `loadjQueryDependentScript` as well?